gravitee-am-identityprovider/gravitee-am-identityprovider-github/src/main/java/io/gravitee/am/identityprovider/github/authentication/GithubAuthenticationProvider.java

```diff
@@ -54,6 +56,7 @@ public class GithubAuthenticationProvider extends AbstractSocialAuthenticationPr
     private static final String CLIENT_SECRET = "client_secret";
     private static final String REDIRECT_URI = "redirect_uri";
     private static final String CODE = "code";
+    private static final String ACCESS_TOKEN = "access_token";
```

Comment:
Could be nice to move this constant up to the SocialAuthProvider interface to avoid duplication with [AbstractOIDC one](https://github.com/gravitee-io/gravitee-access-management/blob/b01b5a916e2c533b63949c8ba97303ce95e4396f/gravitee-am-identityprovider/gravitee-am-identityprovider-common/src/main/java/io/gravitee/am/identityprovider/common/oauth2/authentication/AbstractOpenIDConnectAuthenticationProvider.java#L77).